Klipper_Files/Extra module/ercf.py

```diff
@@ -959,6 +962,13 @@ def _load_to_end_of_bowden(self, length):
             return
 
     def _home_to_extruder(self, length, step):
+        if not self.toolhead_use_stallguard:
```

Comment:
```
        if (self.is_paused):
            return
```
else a previous failure while _do_load is ignored and the whole print is cancelled when the home to extruder move fails.